community/consistency-check/src/main/java/org/neo4j/consistency/checking/full/StoreProcessor.java

```diff
@@ -145,4 +205,61 @@ public void processSchema( RecordStore<DynamicRecord> store, DynamicRecord schem
             checkSchema( RecordType.SCHEMA, store, schema, schemaRecordCheck );
         }
     }
+
+    public <R extends AbstractBaseRecord> void applyFilteredParallel( RecordStore<R> store,
```

Comment:
This method looks very similar to `ParallelRecordScanner#scan`. Could those two be unified?